Signal/src/ViewControllers/HomeView/HomeViewCell.m

```diff
@@ -345,6 +326,42 @@ - (UIFont *)nameSecondaryFont
     return [UIFont ows_dynamicTypeFootnoteFont];
 }
 
+// A simple function to scale dimensions to reflect dynamic type.  Given a value
+// we lerp it larger linearly to reflect size of dynamic type relative to a
+// reference value for default dynamic type sizes.
+//
+// * We _NEVER_ scale values down.
```

Comment:
> I guess this means spacing will maybe look a little "too large" for people who scale down their font size,

I did it this way because the same margins work for "small" and "default" text - at least to my eye.  I'll be confirming with Myles tomorrow.